tests/test_prettytable.py

```diff
@@ -94,6 +94,60 @@ def mix_prettytable():
     return mix
 
 
+class TestNoneOption:
+    def test_none_char_valid_option(self):
+        PrettyTable(["Field 1", "Field 2", "Field 3"], none_format="")
+
+    def test_none_char_invalid_option(self):
+        with pytest.raises(Exception) as exc:
+            PrettyTable(["Field 1", "Field 2", "Field 3"], none_format=2)
+            assert "must be a string." in exc.value
+
+    def test_no_value_replace_none(self):
+        t = PrettyTable(["Field 1", "Field 2", "Field 3"])
+        t.add_row(["value 1", None, "value 2"])
+        assert (
+            """
++---------+---------+---------+
+| Field 1 | Field 2 | Field 3 |
++---------+---------+---------+
+| value 1 |   None  | value 2 |
++---------+---------+---------+
+""".strip()
+            == t.get_string().strip()
+        )
+
+    def test_replace_none_all(self):
+        t = PrettyTable(["Field 1", "Field 2", "Field 3"], none_format="N/A")
+        t.add_row(["value 1", None, None])
```

Comment:
Let's include both a `None` and `"None"` string here to demonstrate them both being replaced by `"N/A"`:

```suggestion
        t.add_row(["value 1", None, "None"])
```